From 77375b5d0c25c5619670fac2ee7dc6d3eaa4bd06 Mon Sep 17 00:00:00 2001
From: AB <b@doubi.fun>
Date: Thu, 22 Jun 2023 09:42:05 +0000
Subject: [PATCH] add dns_config to expectsLargerRequests list

---
 internal/home/middlewares.go | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/internal/home/middlewares.go b/internal/home/middlewares.go
index 5ad02ee0..f1feb45f 100644
--- a/internal/home/middlewares.go
+++ b/internal/home/middlewares.go
@@ -45,7 +45,8 @@ func expectsLargerRequests(r *http.Request) (ok bool) {
 
 	p := r.URL.Path
 	return p == "/control/access/set" ||
-		p == "/control/filtering/set_rules"
+		p == "/control/filtering/set_rules" ||
+		p == "/control/dns_config"
 }
 
 // limitRequestBody wraps underlying handler h, making it's request's body Read
-- 
2.39.2

